mappings/net/minecraft/item/ItemPlacementContext.mapping

```diff
@@ -1,15 +1,16 @@
 CLASS baf net/minecraft/item/ItemPlacementContext
-	FIELD a canReplaceHitBlock Z
-	FIELD g placedPos Lev;
+	FIELD a canReplaceExisting Z
```

Comment:
`existingReplaceable`? Consider a name that doesn't clash with the method name if possible.

Reply:
Why does it need to differ from the method? The field is private.

Reply:
It is fine. Just in case there is a setter, but there is not.